concordium-contracts-common/src/schema.rs

```diff
@@ -1507,6 +1723,23 @@ mod tests {
         assert_eq!(serial_deserial(&f6), Ok(f6));
     }
 
+    #[test]
+    fn test_function_v3_serial_deserial_is_id() {
+        let f1 = FunctionV3 {
+            index:        15,
```

Comment:
I think you could remove `index` from the struct, but if not it would be easier to read:
```suggestion
            index:        0b1111,
```